### ISSUE TITLE
Bug: name 'nof_nan_errors' is not defined

### ISSUE THREAD

**Issue by SLornieCYC** [data-to-insight/csc-validator-be-903#545]
**Describe the bug**
Inclusion of #536 has broken live tool?

**Proposed fix**
Think the issue is with this block of code which tries to check and return `nof_nan_errors` rather than `nof_nans`:

```
nof_errors = len(values)
# select out only the valid values, that is remove all nans.
values = [i for i, not_nan in zip(values, pd.notna(values)) if not_nan]
nof_nans = nof_errors - len(values)
if nof_nan_errors != 0:
  logger.warning(f"{error.code} returned {nof_nan_errors} NaNs! "
                           + f"output: {str(values)}")
```

**Comment by dezog** [comment]
Thanks for this stephen - getting a fix out ASAP!

**Comment by dezog** [comment]
fixed as of 360cfc602987bddf8db6cb3548da264b42c8c135
